server/helper/validationArgenissEmail.js

```diff
@@ -0,0 +1,12 @@
+function checkArgenissFormatEmail(email){
+    var errorMessage = "";
+    const domain = "@argeniss.com";
+
```

Comment:
Please use regular expressions to validate email, like this one: \S+@\S+
or another more complex. We can also add "@argeniss" to the regex to validate argeniss emails.
This would simplify a lot the code.